### ISSUE TITLE
The data display page displays incorrect content

### ISSUE THREAD

**Issue by foriLLL** [gems-uff/merge-nature#18]
When trying to get a few chunks, I find that some pages are mulfunctioning, e.g the 9th. chunk on this [page](https://merge-nature.netlify.app/520146/beam/659b3e0e0bd4fa0a3ccadf4f50101ecac5405e0c/659b3e0e0bd4fa0a3ccadf4f50101ecac5405e0c).  

It will be of great help if you can fix this or if I can simply get the data.

Tnanks.

<img width="1213" alt="image" src="https://github.com/gems-uff/merge-nature/assets/44702632/723d2bd8-839d-4752-9570-01347dfad416.png">

**Comment by gleiph** [comment]
@foriLLL , thanks for reporting this problem. I will work on this issue as soon as possible. Did you find any other page with the same problem? Thanks,
Gleiph

**Comment by foriLLL** [comment]
@gleiph , yeah actually there are some similar problems and a few other string concatenation problems I believe.

For example, in [hudson-cce13](https://merge-nature.netlify.app/252276/hudson/cce1364f0b52f1b1f90d8350ef64f0a0c365de21/cce1364f0b52f1b1f90d8350ef64f0a0c365de21), similar problem happened again on the 16th chunk:

<img width="1271" alt="image" src="https://github.com/gems-uff/merge-nature/assets/44702632/c3575bf4-dcc3-438f-9b6b-77fe590f114d.png">

and in [hudson-2c4cc](https://merge-nature.netlify.app/252276/hudson/2c4cc77d33e836d4b0f8bda7cc575bec8877d9e6/2c4cc77d33e836d4b0f8bda7cc575bec8877d9e6), the 2nd chunk seem to have a string concat problem?

<img width="1275" alt="image" src="https://github.com/gems-uff/merge-nature/assets/44702632/f604ad93-8efd-4cff-b1d2-a70d5c0c0fed.png">

Happy if these can be helpful.
